source/beamline/IDEAS/IDEASBeamline.h

```diff
@@ -218,8 +208,20 @@ protected slots:
 	CLSSIS3820Scaler *scaler_;
 
 	IDEASKETEKDetector *ketek_;
+	AMControl *ketekPeakingTime_;
```

Comment:
That might be from a different pull request.  Maybe we'll need to make sure that code is merged in before a final review.